search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -85,23 +89,22 @@ function displayTopResultsForCurrentTrend(countryCode) {
   }
 }
 
-
 /** 
- * Displays the sentiment chart in a country for current search trend on modal. 
+ * Displays sentiment chart of search results for the current country on the modal.
  * @param {string} countryCode Two letter country code for selected country.
  */
 function displaySentimentChartForCurrentTrend(countryCode) {
-  let topic = getCurrentTrend;
-  let topicData = getCurrentCustomSearchData();
-  let date = new Date(topicData.timestamp);
+  // let topic = getCurrentTrend();
+  let dataByCountry = getCurrentCustomSearchData().dataByCountry;
+  let countryData = dataByCountry.filter(data => data.country === countryCode);
```

Comment:
You don't use dataByCountry anywhere else. Consider combining into one variable and chaining the filter.

Reply:
+1